SECURITY.md

```diff
@@ -1,34 +1,93 @@
 # Security policy
 
-## Supported versions
+## Warning to users
+
+PowSyBl is a library dedicated to electrical grid modelling and simulation. As such, it includes data importers to feed the calculations. Please make sure your data comes from a safe source.
 
-The available compatible releases are listed in [powsybl-dependencies](https://github.com/powsybl/powsybl-dependencies). In case of vulnerability fix, the main branches are first fixed. The fix will be available for the next release train. In case of vulnerability of high or critical severity, the most recent version is patched.
+PowSyBl also provides a persistent IIDM implementation by offering the possibility to connect to an external PostgreSQL database. If you use this feature, please make sure your login and password are secure and safely stored.
 
 ## Reporting a vulnerability
 
-Please report any security vulnerability, suspected or confirmed, to the Technical Steering Committee [powsybl-tsc@lists.lfenergy.org](mailto:powsybl-tsc@lists.lfenergy.org). You can also report totally privately to the Chair of TSC [Anne Tilloy](anne.tilloy@rte-france.com) for high severity vulnerabilities. If the issue is confirmed, we will inform the library users and release a patch as soon as possible depending on the complexity.
+Please report any **high-severity** security vulnerability, suspected or confirmed, to the Technical Steering Committee [powsybl-tsc@lists.lfenergy.org](mailto:powsybl-tsc@lists.lfenergy.org).
```

Comment:
This mailing list is public, I have read that we have to give a totally private way to report vulnerabilities.

Reply:
Ok then I think we have to give several names instead of just one or to create a mailing list with TSC members, in order to ensure the mentioned deadline of response (or any deadline in fact).
@annetill do you have rights to create a new `@lists.lfenergy.org` including only TSC members for example? I don't think I can do it.